tests/openadapt/test_scrub.py

```diff
@@ -19,16 +23,137 @@ def _hex_to_rgb(hex_color: int) -> tuple[int, int, int]:
+    image.save(enhanced_image_path, dpi=(300, 300))
+
+    image_read = cv2.imread(enhanced_image_path)  # pylint: disable=no-member
+    os.remove(enhanced_image_path)
+
+    # Develop contrast for better OCR results with Tesseract
+    # Reference: https://stackoverflow.com/questions/58314275/how-to-make-image-more-contrast-grayscale-then-get-all-characters-exactly-with # pylint: disable=line-too-long # noqa: E501
+
+    gray_image = cv2.cvtColor(  # pylint: disable=no-member
+        image_read, cv2.COLOR_BGR2GRAY  # pylint: disable=no-member
+    )  # developing contrast for better OCR results with Tesseract
+    thr_image = cv2.adaptiveThreshold(  # pylint: disable=no-member
+        gray_image,
+        255,
+        cv2.ADAPTIVE_THRESH_MEAN_C,  # pylint: disable=no-member
+        cv2.THRESH_BINARY_INV,  # pylint: disable=no-member
+        23,
+        100,
+    )
+    bnt_image = cv2.bitwise_not(thr_image)  # pylint: disable=no-member
+    cv2.imwrite(enhanced_image_path, bnt_image)  # pylint: disable=no-member
+
+
+def test_emr_image() -> None:
+    """Test to verify that an EMR scrubbed image does not contain any PII/PHI."""
+    test_image_path = "assets/test_emr_image.png"
+    test_image_path_300 = "assets/test_emr_image_300.png"
+    scrubbed_image_path = test_image_path[:-4] + "_scrubbed.png"
+
+    _enhance_image_for_ocr(test_image_path, test_image_path_300)
+    # original_image_text = (
+    #     pytesseract.image_to_string(Image.open(test_image_path_300))
+    # )
+    os.remove(test_image_path_300)
+
+    # Preparing for Scrubbing
+    with open(test_image_path, "rb") as file:
+        test_image_data = file.read()
+
+    # Convert image data to PIL Image object
+    test_image = Image.open(BytesIO(test_image_data))
+
+    # Scrub the image
+    scrubbed_image = scrub.scrub_image(test_image)
+
+    # Save the scrubbed image data to a file
+    scrubbed_image.save(scrubbed_image_path)
+
+    _enhance_image_for_ocr(scrubbed_image_path, test_image_path_300)
+    scrubbed_image_text = pytesseract.image_to_string(Image.open(test_image_path_300))
+    os.remove(test_image_path_300)
+    test_image.close()
+    os.remove(scrubbed_image_path)
+
+    # try:
+    # Use Cape to detect PII/PHI in the scrubbed image ocr text
+    resp = requests.post(  # pylint: disable=missing-timeout
+        "https://api.capeprivacy.com/v1/privacy/deidentify/text",
+        headers={
+            "Authorization": (
+                f"Bearer {config.CAPE_API_KEY}"  # pylint: disable=no-member
+            )
+        },
+        json={
+            "content": scrubbed_image_text,
+        },
+    )
+    cape_response = resp.json()
+
+    if (
+        "entities" not in cape_response.keys()
+        or "content" not in cape_response.keys()
+        or cape_response is None
+    ):
+        pytest.skip(reason="Cape failed to return entities and content")
+
+    detect_entities = cape_response.get("entities")
+    important_entities = [
```

Comment:
Important Entities (from Cape) are those which are similar to those which are detected by Presidio.